ygen/enumgen.go

```diff
@@ -224,11 +224,23 @@ func (s *enumSet) identityBaseKey(i *yang.Identity) string {
 	return fmt.Sprintf("%s/%s", definingModYANGName, i.Name)
 }
 
+// nodePath returns the full path of the node from the module name.
+func nodePath(n yang.Node) string {
+	path := n.NName()
+	for n.ParentNode() != nil {
```

Comment:
I think this can be written in a more compact form, can't it?

```golang
for path := n.NName(); n.ParentNode() != nil; n = n.ParentNode() {
   path = n.NName() + "/" + path
}
return path
```